tools/update_crates.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
```

Comment:
Can you document this (and the Cargo/crate story in general) in https://github.com/envoyproxy/envoy/blob/master/bazel/EXTERNAL_DEPS.md?

Reply:
Does it belong there? `EXTERNAL_DEPS.md` is for Envoy's direct dependencies, whereas what we do here is specifically tailored for generating Proxy-Wasm extensions for the test data, and doesn't apply to other dependencies.

Reply:
Maybe create a `bazel/WASM.md` or `bazel/RUST.md`? Just somewhere so that we have a place to orient folks and explain the Cargo/Raze/Rust/Wasm test story.

Reply:
Added. Let me know if that works for you.